frontend/components/draft/lesson/block_tile.tsx

```
@@ -0,0 +1,134 @@
+  totalBlocks,
+  moveBlockUp,
+  moveBlockDown,
+  setBlock,
+  deleteBlock,
+}: {
+  block: Block;
+  index: number;
+  totalBlocks: number;
+  moveBlockUp: (index: number) => void;
+  moveBlockDown: (index: number) => void;
+  setBlock: (index: number) => (block: any) => void;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)